src/plugins/data/common/search/aggs/param_types/field.ts

```diff
@@ -43,6 +43,7 @@ export class FieldParamType extends BaseParamType {
 
     this.filterFieldTypes = config.filterFieldTypes || '*';
     this.onlyAggregatable = config.onlyAggregatable !== false;
+    this.scriptable = config.scriptable !== false;
```

Comment:
Strange... we had this in types and a default `scripted = true`, but it was never set from `config.scriptable` 🤔